src/openssl/algorithm.cr

```diff
@@ -0,0 +1,34 @@
+require "openssl"
+
+module OpenSSL
+  # Indicates the possible hash algorithms for `OpenSSL` digest operations.
+  enum Algorithm
+    MD4
+    MD5
+    RIPEMD160
+    SHA1
+    SHA224
+    SHA256
+    SHA384
+    SHA512
+
+    # The internal bindings to the `LibCrypto` digest operations sometimes require a hash algorithm
+    # implementation to be passed as one of the arguments.
+    #
+    # This method will return the appropriate equivalent hash algorithm that corresponds to the
```

Comment:
"This method ..." is redundant. It's obvious that `to_evp` is a method and thus saying it's a method is unnecessary.
```suggestion
    # Returns the appropriate equivalent hash algorithm that corresponds to the
```